event-processing/obfuscation/tests/unit/test-helper.ts

```diff
@@ -40,9 +40,9 @@ export class TestHelper {
         event_name: 'AUTHENTICATION_ATTEMPT',
         component_id: 'AUTH',
         user: {
-            transaction_id: ObfuscationService.obfuscate('a52f6f87', 'secret-1-value'),
-            email: ObfuscationService.obfuscate('foo@bar.com', 'secret-1-value'),
-            phone: ObfuscationService.obfuscate('07711223344', 'secret-1-value'),
+            transaction_id: ObfuscationService.obfuscateField('a52f6f87', 'secret-1-value'),
+            email: ObfuscationService.obfuscateField('foo@bar.com', 'secret-1-value'),
+            phone: ObfuscationService.obfuscateField('07711223344', 'secret-1-value'),
```

Comment:
can you please also add an unexpected field, say "foo", to check it is obfuscated as well

Reply:
@phantas do you agree with this or would you prefer adding an unexpected field for posterity?

Reply:
I am ok with that